patch_subset/codepoint_map.h

```diff
@@ -0,0 +1,35 @@
+#ifndef PATCH_SUBSET_CODEPOINT_MAP_H_
+#define PATCH_SUBSET_CODEPOINT_MAP_H_
+
+#include <unordered_map>
+#include <vector>
+
+#include "common/status.h"
+#include "hb.h"
+#include "patch_subset/patch_subset.pb.h"
+
+namespace patch_subset {
+
+class CodepointMap {
```

Comment:
describing the data this manages, what encode/decode means, and how the proto representation works would help. It's comprehensible but I felt like I spent more time than I should have looking up things like Encode ... what's Encode? Oh it's Remap... what's Remap?